serialization/bson.ksy

```diff
@@ -0,0 +1,180 @@
+meta:
+  id: bson
+  file-extension: bson
+  endian: le
+  license: CC0
```

Comment:
By the way, [SPDX says](https://spdx.org/licenses/CC0-1.0.html) that this should be marked up as `CC0-1.0`, not just `CC0`.